app/client/aclui/ACLFormulaEditor.ts

```diff
@@ -133,6 +133,9 @@ const cssConditionInputAce = styled('div', `
   & .ace-chrome, & .ace-dracula {
     background-color: ${theme.accessRulesFormulaEditorBg};
   }
+  & .ace-chrome .ace_print-margin, & .ace-dracula .ace_print-margin {
+    width: 0px;
+  }
```

Comment:
```suggestion
  &:not(:focus-within) .ace_print-margin {
    width: 0px;
  }
```

This hides the indicator line when the editor isn't focused. Feels like a good compromise. Wdyt?

Reply:
Well, if you prefer it's good to me, I just does not understand why this separator line in the middle of the editor it's important. It's look like a visual glitch to me.

Reply:
Or it have to be closer to the indicators

Reply:
`right: 5px !important;
    left: unset !important;`

Reply:
```suggestion
  & .ace-chrome .ace_print-margin, & .ace-dracula .ace_print-margin {
    right: 5px !important; 
    left: unset !important;
  }
```

Reply:
It's probably a good idea to keep it where it is while the editor has focus. It's consistent with the formula editor elsewhere in Grist, and it's common in Python code to limit lines to ~80 characters, which is where the line starts.